### PR TITLE
feat(keys): support scoped keys

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes https://github.com/mozilla/fxa-oauth-server/issues/484
Connects to mozilla/fxa#217

### CODE REVIEW

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: ff45c66731628fd5cdb04949b7260d956b95558a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-oauth-server

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: ff45c66731628fd5cdb04949b7260d956b95558a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-oauth-server

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: ff45c66731628fd5cdb04949b7260d956b95558a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-oauth-server

---
This message was auto-generated by https://gitcop.com

File: lib/routes/token.js [hunk 1]
```diff
@@ -160,7 +160,8 @@ module.exports = {
       scope: validators.scope.required().allow(''),
       token_type: Joi.string().valid('bearer').required(),
       expires_in: Joi.number().max(MAX_TTL_S).required(),
-      auth_at: Joi.number()
+      auth_at: Joi.number(),
+      derivedKeyBundle: Joi.string().optional()
```
Comment (rfk) [inline lib/routes/token.js]: In [1] the name of this field is "keys_jwe", by symmetry with the "keys_jwk" parameter passed in at the start of the dance.

[1] https://docs.google.com/document/d/1IvQJFEBFz0PnL4uVlIvt8fBS_IPwSK-avK0BRIHucxQ/edit#

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: 8f786356ef0c6617d657d23292647d1ef75ea290
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-oauth-server

---
This message was auto-generated by https://gitcop.com

**Pull request closed without merging**